scripts/migration-drift/check.ts

```
@@ -0,0 +1,513 @@
+    process.exit(2);
+  }
+
+  let report: MigrationReport;
+  try {
+    report = buildReport(applied, repoMigrations());
+  } catch (err) {
+    console.error((err as Error).message);
+    process.exit(2);
+  }
+
+  console.log(formatReport(report));
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)